README.md

```diff
@@ -87,5 +87,10 @@ docker build --target development --tag todo-app:dev .
 docker build --target production --tag todo-app:prod .
 docker run -p 5001:5001 --env-file .env  --mount=type=bind,source={source on your machine},target="/todo-app"  -d  todo-app
 
+docker build --target test --tag my-test-image .
```

Comment:
A little bit of context for these would be nice, but they do work out of the box!